src/Compiler/CompilerOptions.cpp

```diff
@@ -711,6 +712,16 @@ static llvm::cl::opt<bool, true> enable_bound_check("enable-bound-check",
     llvm::cl::location(enableBoundCheck), llvm::cl::init(false),
     llvm::cl::cat(OnnxMlirOptions));
 
+static llvm::cl::list<std::string, std::vector<std::string>>
+    decomposeOpsInONNXOpt("decompose-op-in-onnx",
+        llvm::cl::desc("Specify ONNX operations to decompose.\n"
+                       "Supported Ops - HardSwish"),
+        llvm::cl::value_desc("ONNX operation to decompose"),
+        llvm::cl::location(decomposeOpsInONNX), 
+        llvm::cl::cat(OnnxMlirOptions),
```

Comment:
This should be `OnnxMlirCommonOptions` if you want to use it in both `onnx-mlir` and `onnx-mlir-opt` commands.